lib/utils/testLauncherHelper.js

```diff
@@ -80,7 +80,7 @@ function readLauncherConfig() {
  */
```

Comment:
Please update description, so its clear that configB is returned unless its property nullable

Reply:
updated